.github/workflows/create-block.yml

```diff
@@ -0,0 +1,28 @@
+
+
+name: Create Block
+
+on:
+  push:
+    paths:
+    - 'packages/**'
```

Comment:
I'm still unsure of what would be important here, maybe `package*.json` to cover the case when dependencies change. Although it would probably mean that one of the packages changed as well.

Reply:
I initially thought about only adding @wordpress dependencies for create-block but it has none. 🙈 

In this case probably all of https://github.com/WordPress/gutenberg/blob/cb10e2d6e249000858da4eb3fc07cc44c832eddb/packages/scripts/package.json#L36-L42, `packages/scripts/package.json` and `packages/create-block`?

Reply:
There is also `@wordpress/browserlist-config` or `@wordpress/element` that is used by Babel. I said, it might be tricky to identify all dependencies used :)

Reply:
Once we add new configs, like `@wordpress/styleling-config` as proposed by @ntwb in #22777, we would have to update this list. I would personally prefer to extend the list of excluded files rather than allow list.

Reply:
Let's use the current list and see how it goes.